examples/datasets/foods1.ndjson

```diff
@@ -1,27 +0,0 @@
-{"category":"vegetables","calories":45,"fats_g":0.5,"sugars_g":2}
```

Comment:
Was this intentional?

Reply:
Yes. The files `foods1.ndjson` and `foods2.ndjson` are created in polars/polars-lazy/src/tests/mod.rs

Reply:
Ah, right. :+1: